languages/go/exercises/concept/time/.docs/introduction.md

```diff
@@ -0,0 +1,22 @@
+## Time
+
+In Go, functionality for working with times is provided by the [time package][time_pkg]. The types and methods in this package allow us to manipulate times, get the current time, determine elapsed time, parse times from strings, and more.
```

Comment:
This is a nice introduction! One small nit, the introduction document should preferrably not use links (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd)):

> Links should be used sparingly, if at all. While a link explaining a complex topic like recursion might be useful, for most concepts the links will provide more information than needed so explaining things concisely inline should be the aim.

The reason for this is that we want students to figure things out by themselves. To help guide students in the right direction, we use two things:

1. The `hints.md` file, which we _do_ expect to link to external resources
1. Highlight packages/modules as source code, signifying that the highlighted term is exactly the right name which can be used to search for.

So maybe do something like this:

```suggestion
In Go, functionality for working with times is provided by the `time` package. The types and methods in this package allow us to manipulate times, get the current time, determine elapsed time, parse times from strings, and more.
```